dependencies/current/dealii/install.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+# This file is part of 4C multiphysics licensed under the
+# GNU Lesser General Public License v3.0 or later.
+#
+# See the LICENSE.md file in the top-level for license information.
+#
+# SPDX-License-Identifier: LGPL-3.0-or-later
+
+# Install deal.II
+# Call with
+# ./install.sh /path/to/install/dir
+
+# Exit the script at the first failure
+set -e
+
+INSTALL_DIR="$1"
+# Number of procs for building (default 4)
+NPROCS=${NPROCS:=4}
+VERSION="6890ca740b6e51530ebb7ae7e4932977010ef2be" # version 9.6.2
+#CHECKSUM=""
+
+
+CMAKE_COMMAND=cmake
+
+git clone https://github.com/dealii/dealii.git
+cd dealii
+git checkout $VERSION
+mkdir build
+cd build
+
+# Install p4est as part of deal.II
+wget --no-verbose https://p4est.github.io/release/p4est-2.8.tar.gz
+../doc/external-libs/p4est-setup.sh p4est-2.8.tar.gz $INSTALL_DIR
```

Comment:
This script internally untars the archive, which gives some ownership warnings. The resulting image worked for me locally, so I did not try to fix that.